_topics/en/eclipse-attacks.md

```diff
@@ -77,6 +77,9 @@ optech_mentions:
   - title: LND 0.14.0-beta includes additional eclipse attack protection by sharing block headers
     url: /en/newsletters/2021/11/24/#lnd-0-14-0-beta
 
+  - title: "Bitcoin Core #27213 will attempt to open and maintain one connection on each reachable network"
```

Comment:
@harding would it also be a valuable topic entry for https://bitcoinops.org/en/topics/anonymity-networks/ ?

Reply:
I thought about that but decided against it.  This is really about eclipse protection and preventing partitions, which takes advantage of those networks when they're available but isn't special to those networks (e.g. it also works on ipv4 vs ipv6).